<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>info_imagen</class>
 <widget class="QDialog" name="info_imagen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="regresar">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgba(0,0,0,1);
font: 10pt &quot;Romantic&quot;;</string>
   </property>
   <property name="text">
    <string>Regresar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Romantic&quot;;
border: 2px solid rgba(0,0,0,1);</string>
   </property>
   <property name="text">
    <string>  INFORMACIÃ“N DE IMAGEN </string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>321</width>
     <height>164</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Romantic&quot;;
text-decoration: underline;</string>
      </property>
      <property name="text">
       <string>Sexo paciente:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="mode">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Romantic&quot;;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="pat_sex">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Romantic&quot;;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Romantic&quot;;
text-decoration: underline;</string>
      </property>
      <property name="text">
       <string>Modo:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="minimumSize">
       <size>
        <width>182</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Romantic&quot;;
text-decoration: underline;</string>
      </property>
      <property name="text">
       <string>Fecha:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Romantic&quot;;
text-decoration: underline;</string>
      </property>
      <property name="text">
       <string>Parte del cuerpo examinada: </string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Romantic&quot;;
text-decoration: underline;</string>
      </property>
      <property name="text">
       <string>Peso paciente: </string>
      </property>
     </widget>
    </item>
    <item row="8" column="1" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="peso_pac">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Romantic&quot;;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="body_part">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Romantic&quot;;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="date">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Romantic&quot;;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>-10</y>
     <width>451</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Imagenes/info.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_7</zorder>
  <zorder>regresar</zorder>
  <zorder>label</zorder>
  <zorder>gridLayoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
